### ISSUE TITLE
should loss function for vae use binary_cross_entropy?

### ISSUE THREAD

**Issue by ethanjyx** [SherlockLiao/pytorch-beginner#34]
https://github.com/L1aoXingyu/pytorch-beginner/blob/052ed0d4e3d3959e86c88737fd133ef5d0180acc/08-AutoEncoder/Variational_autoencoder.py#L75

this line here it assigned to variable called BCE, but uses MSE instead
